Gemfile

```diff
@@ -23,4 +23,8 @@ group(:dev) do
   gem 'guard'
   gem 'guard-rspec'
   gem 'ruby_gntp'
+  gem 'chefspec'
+  gem 'rubocop'
+  gem 'knife-spork'
+  gem 'kitchen-vagrant'
```

Comment:
:development is the magic group that `gem install` will ignore but a vanilla `bundle install` without any args will install by convention.  if you name it `:dev` its a bit useless since it'll install with both a default `gem install` and a `bundle install` so you might as well just make it a normal dependency unless you're going to use `bundle install --without dev` explicitly for some reason.

looks like you can rename it though and use `:dev` instead of `:development`.

i suspect what is intended is using `:development` but it got copypasta'd from some other project that remapped the `:development_group` to `:dev` but that line wasn't copied...